### DISCUSSION TITLE
可以交接專案負責人嗎?

### DISCUSSION THREAD

**Discussion by bestian** [3dw/auto20-next#120] [Q&A] [answered]
(此事不急，只是先寫出來)

由於自學2.0涉及個資收集

之前 @bestian 有過社會事件, 社會形象較為不利於專案的宣傳
且精神狀態仍在休養中, 使用電腦的時間很少. 且起伏的過程對維護專案的穩定程度不利.

想用非同步的方式交接掉自學2.0專案負責人
轉為技術顧問和工程師.

專案負責人的權責:

1. auto20-admin這個repo的讀寫權限
2. 可以密件副本發信給所有用戶的權限(請慎重使用)
3. Firebase Database的管理權限
4. 刪除所有以Facebook登人的用戶的權限(技術: node.js, 交件文件由@bestian準備)
5. 引導及宣傳本專案的權限(不特別宣傳也沒關係)
6. 在共筆區留下管理員工作流程計畫的義務 [https://github.com/3dw/auto20-admin/issues/2](https://github.com/3dw/auto20-next/issues/31)
7. 年度統計Google分析報告給會員的工作

不確定
@3dw/educator 
中任何成員有無意願接手自學2.0專案負責人?

**Comment by bestian** [comment]
簡報檔
https://docs.google.com/presentation/d/1mok1-N8zmTUKukIdtetemBejXgJdBcCL_U6iXBFWJyQ/edit?usp=sharing

**Reply by leechiuhui** [reply, marked as answer]
宗浩老師，您辛苦了 。
這一陣子在爬auto20-next程式的過程當中，除了自覺進步很多以外，也真的覺得整個程式邏輯非常的細微 ，是需要花很多歷史的沈澱累積和腦汁的。
看了這個群組的watching成員名單，除了昨天我為了模擬deploy我自己加入另一個自己在用的學生測試帳號SJ以外，還有另外三位成員，所以如果過一陣子，都「沒有人」要接的話，那就套一句g0v的精神口號,我可以當那個「沒有人」。
以向您和3DW的所有人致敬。

Drago敬上🙏

**Reply by bestian** [reply]
> 宗浩老師，您辛苦了 。 這一陣子在爬auto20-next程式的過程當中，除了自覺進步很多以外，也真的覺得整個程式邏輯非常的細微 ，是需要花很多歷史的沈澱累積和腦汁的。 看了這個群組的watching成員名單，除了昨天我為了模擬deploy我自己加入另一個自己在用的學生測試帳號SJ以外，還有另外三位成員，所以如果過一陣子，都「沒有人」要接的話，那就套一句g0v的精神口號,我可以當那個「沒有人」。 以向您和3DW的所有人致敬。
> 
> Drago敬上🙏

非常感謝.

我先把相關權限(Google 分析, Firebase擁有者, auto20-admin)都開了.

https://we.alearn.org.tw 也歡迎改動

Deploy的方法如下：

https://github.com/3dw/auto20-next?tab=readme-ov-file#%E5%BB%BA%E6%A7%8B%E6%9B%B4%E6%96%B0%E4%B8%A6%E4%B8%8A%E5%82%B3%E8%87%B3%E7%B6%B2%E8%B7%AF


另，
交接資料寫在auto20-admin專案的README.md中.
https://github.com/3dw/auto20-next?tab=readme-ov-file#%E5%BB%BA%E6%A7%8B%E6%9B%B4%E6%96%B0%E4%B8%A6%E4%B8%8A%E5%82%B3%E8%87%B3%E7%B6%B2%E8%B7%AF


目前的版本有點簡略, 有疑問麻煩再告訴我(寫在auto20-admin專案的issues中), 我來補充.
